packages/next-swc/crates/napi/npm/linux-arm64-gnu/install.js

```diff
@@ -0,0 +1,8 @@
+const { glibcVersionRuntime } = process.report.getReport().header;
+const { name } = require('./package.json');
+
+if (glibcVersionRuntime && name.endsWith('-gnu') || !glibcVersionRuntime && !name.endsWith('-gnu')) {
```

Comment:
Should this contain `-gnu`? It looks like this is just a version number on Ubuntu:
```sh
> process.report.getReport().header.glibcVersionRuntime
'2.23'
```